services-toolkit/explanation/four-levels-of-service-consumption.hbs.md

```diff
@@ -0,0 +1,69 @@
+
+The second limitation is that all resources have to be in the same namespace, which is ok, but it is a bit limiting.
+
+The third limitation is that the service binding has to know detailed and specific information about the service resource, including its name, namespace and API Group/Version and Kind. This is not a particularly nice or clear separation of concerns as it introduces tight coupling between app teams, who create the application workloads, and ops teams, who create the service resources.
+
+## Level 2 - ResourceClaims
+
+In order to address some of these limitations, to coincide with the release of TAP 1.0 services toolkit shipped with a capability known as ResourceClaims.
+
+![Diagram shows level 2 of service consumption in TAP](../../images/stk-4-levels-2.png)
+
+Let's imagine the same setup as before, but this time we want the service resource to be in a separate namespace from the Knative service. Unfortunately this is not possible with direct bindings. If you look at the [schema](https://github.com/servicebinding/spec#resource-type-schema-1) for ServiceBinding you will note that there is no option to configure a `namespace` for the service. This is where ResourceClaims enter the picture.
+
+ResourceClaims allow you to claim a bindable service resource that exists in another namespace, and to then bind the application workload to the ResourceClaim instead of to the service resource directly. Note that because we are now essentially jumping across namespace tenancy boundaries, this claiming across namespaces is only permitted if a corresponding ResourceClaimPolicy has been created.  
+
+The big advantage of level 2 over level 1 is that now the application workload and the service resource do not have to exist in the same namespace. This helps to promote a better separation of concerns. It is now possible for apps teams and ops teams to manage the lifecycles of apps and services inderpendently, without stepping on each others' toes.
+
+However it's still not an ideal solution, and some of the limitations from level 1 still exist here in level 2. The service resource still has to be bindable and apps teams still need to know the name, namespace and API GVK of the service resource. In addition, ops teams now need to ensure that the service resources actually exist. These resources must be manually provisioned and permitted to be claimed through policy, as otherwise ResourceClaims created by the apps teams will just sit around indefinitely in a pending state.
```

Comment:
**nit:** You mention _"ops teams NOW need to ensure that the service resources actually exist"_. I don't think this is any different from level 1 or 3.